enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -3319,6 +3319,13 @@ func (p *PebbleCache) Stop() error {
 	}
 	log.Infof("Pebble Cache [%s]: waitgroups finished", p.name)
 
+	// Flushed db after all waitgroups finished to reduce the change of lost
```

Comment:
Can you call p.flushPartitionMetadata() as well? Would be good if we minimize any lost updates there as well.

Reply:
Done.